repository/spark/docs/beta1/monitoring.md

```diff
@@ -16,77 +16,10 @@ If you don't already have the `prometheus-operator` installed on your Kubernetes
 the [quick start guide](https://github.com/coreos/prometheus-operator#quickstart).
 
 ##### Metrics configuration
-Metrics configuration used by KUDO Spark can be specified by providing the following parameters:
-```bash
-kubectl kudo install spark --instance=spark-operator \
-        -p enableMetrics=true \
-        -p operatorMetricsPort=10254 \
-        -p appMetricsPort=8090 \
-        -p metricsEndpoint=/metrics \
-        -p metricsPrefix="" \
-        -p metricsPollingInterval="5s"
-```
+Metrics reporting is enabled by default. Service endpoints and ServiceMonitor resources are configured to work with Prometheus Operator
+out of the box.
```

Comment:
I think it is worth mentioning somewhere in this section, that `appMetricsPort` should be the same for all submitted Spark Applications in order for metrics to be scraped